src/Command/AddEventCommand.php

```diff
@@ -21,6 +21,7 @@ protected function getEventName(): string
      * @param null|string $payload
      * @return null|string
      * @throws GitLabCIFileException
+     * @throws \TheAentMachine\Exception\MissingEnvironmentVariableException
```

Comment:
`use \TheAentMachine\Exception\MissingEnvironmentVariableException` and then `@throws MissingEnvironmentVariableException`